tests/CoreFX/CoreFX.issues.rsp

```diff
@@ -56,6 +56,8 @@
 -nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_StackOverflowException
 -nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_InnerException
 -nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_ArgumentException
+-nomethod System.Data.Tests.Common.DbConnectionStringBuilderTest.Add_Keyword_Invalid
```

Comment:
I am not able to figure out how this change made the test fail by just looking at the code. Why does it fail now?

Reply:
It is checking that the AE message does not contain the parameter. Now it does.
`                    Assert.True(ex.Message.IndexOf("'" + keyword + "'") == -1);`